src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -97,8 +97,10 @@ export const AdvancedSearchPage = () => {
           </Grid>
 
           <Grid container direction={isLargeScreen ? 'row' : 'column'} gap={2} size={12}>
-            <Grid sx={{ width: 'fit-content' }}>
-              <StyledFilterHeading>{t('search.advanced_search.publishing_period')}</StyledFilterHeading>
+            <Grid component="fieldset" sx={{ width: 'fit-content', border: 'none' }}>
+              <Typography component="legend" sx={{ mb: '0.2rem', fontWeight: 'bold' }}>
```

Comment:
Hvorfor ikke fortsette med samme komponent for heading her som alle andre bruker?
Ser ut som man må legge til støtte for alle props da: `const StyledFilterHeading = styled(Typography)<TypographyProps>(...)`

```suggestion
              <StyledFilterHeading component="legend">
```

Reply:
Kan ikke endre component på en `StyledComponent` virker det som 🤔